### ISSUE TITLE
typos

### ISSUE THREAD

**Issue by spaette** [dmitry-merzlyakov/nledger#26]
I can submit a PR on this.

Can you look over the replacement strings?

Asset
Enumerable
Optimal
Powershell
Quantity
Simulate
Uninstalling
Windows
according
adding
administrative
aligned
amount's
colorization
contains
destructor
efficient
exclusive
expect
explicitly
functionality
implicit
indicator
lengths
necessary
occurred
particular
privileges
privileges
returns
routines
scope
successful
successful
suppresses
value

```
$ grep -nr Assset nledger
nledger/Contrib/test/regress/89233B6D-a.dat:2:    Asssets:Bank:Checking               200.00
nledger/Contrib/test/regress/89233B6D.test:10:94-Jan-02 Salary                A:Assset:Bank:Checking          200          200
nledger/Contrib/test/regress/89233B6D-b.dat:3:    Asssets:Bank:Checking              -100.00
$ grep -nr Enuemrable nledger
nledger/Source/NLedger/Extensibility/SessionExtensions.cs:65:        /// <param name="args">Enuemrable of strings that contains a command with arguments</param>
$ grep -nr Optinal nledger
nledger/Source/NLedger/Utility/Settings/CascadeSettings/Sources/EnvironmentVariablesSettingsSource.cs:26:        /// <param name="namePrefix">Optinal filtering prefix for setting names</param>
$ grep -nr "Powershel " nledger
nledger/Build/NLedgerWix/InstallerScripts.vbs:3:' Shows a warning message that Powershel is not installed
$ grep -nr Qunatity nledger
nledger/Source/NLedger.Tests/BigIntTests.cs:22:    // See Amount.cs for further information about Qunatity Arithmetics in NLedger
$ grep -nr Simluate nledger
nledger/Contrib/Extras/cat.cmd:1:@rem Simluate "cat" command on Windows environment
$ grep -nr Uniinstalling nledger
nledger/Contrib/Install/NLInstall.psm1:127:    Write-Verbose "Uniinstalling NLedger path $($info.Path)"
$ grep -nr Wiindows nledger
nledger/Contrib/Install/NLInstall.psm1:5:    Installs, uninstalls and checks deployment status on any platform (Wiindows, Linux, OSX)
nledger/Contrib/Install/SysPath.psm1:5:    Adds, removes and checks paths in PATH variable on any platform (Wiindows, Linux, OSX)
nledger/Contrib/Install/SysHardLink.psm1:5:    Creates, removes and checks file hard links on any platform (Wiindows, Linux, OSX)
$ grep -nr accorrding nledger
nledger/Source/NLedger.Tests/Amounts/AmountTests.cs:286:            Assert.False(amountA.IsZero);  // The value is rounded to 0.01 accorrding to Commodity precision
$ grep -nr addiing nledger
nledger/get-nledger-tools.ps1:7:        1) Installing and uninstalling NLedger (that means addiing to PATH variable, creating 'ledger' hard link and some other activities);
$ grep -nr adminidstrative nledger
nledger/Contrib/NLManagement/NLSetup.Console.ps1:243:    Default: False. Indicates that the current scope is Application. You must have adminidstrative priviledges.
nledger/Contrib/NLManagement/NLSetup.Console.ps1:290:    Default: False. Indicates that the current scope is Application. You must have adminidstrative priviledges.
$ grep -nr alligned nledger
nledger/Contrib/test/regress/370-period.test:1:;period days should not be alligned
$ grep -nr "amount'ss" nledger
nledger/Source/NLedger/Amounts/Amount.cs:1508:        /// to_string() returns an amount'ss "display value" as a string --
nledger/Contrib/test/regress/25A099C9.dat:495:      to_string() returns an amount'ss "display value" as a string --
$ grep -nr colorizatiion nledger
nledger/Contrib/NLManagement/NLCommon.psm1:192:    Input string that can contain colorizatiion tokens.
$ grep -nr coontains nledger
nledger/get-nledger-tools.ps1:15:    If the current deployment coontains binaries for both platforms (.Net Framework and .Net Core) - installs .Net Framework binaries.
$ grep -nr descructor nledger
nledger/Source/NLedger/Scopus/Report.cs:689:            // [DM] Disposing handlers at the moment when they go out of scope (like invoking a descructor in c++ code)
$ grep -nr efficiemt nledger
nledger/nledger.md:385:you can revert your changes and continue experimenting with it. So, it is some kind of a playground that let you learn NLedger features in easy and efficiemt way.
$ grep -nr excluse nledger
nledger/Source/NLedger/MainApplicationContext.cs:165:        // Request an excluse access to the current thread
$ grep -nr expact nledger
nledger/Source/NLedger/Abstracts/IVirtualConsoleProvider.cs:62:        /// <param name="str">Text to expact</param>
$ grep -nr explicitely nledger
nledger/Source/NLedger.Extensibility.Python.Module/src/ledger/__init__.py:435:        raise Exception("Method 'get_origin' should be defined in derived class if 'origin' value is not specified explicitely.")
nledger/Source/NLedger.Extensibility.Python.Module/src/ledger/__init__.py:2751:            return Value.to_value(self.origin.ExchangeCommodities(commodities, False, DateTime())) # default(DateTime) caused PythonNet issued, so arguments are populated explicitely
nledger/Source/NLedger.Extensibility.Python.Module/build.ps1:8:    If build parameters are not specified explicitely, it discovers necessary information on the current environment.
nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs:64:                        // it is necessary to run it explicitely (unittest.main(...)) specifying a test file name as a parameter
nledger/Contrib/Python/GetPythonEnvironment.ps1:406:- If 'path' parameter is specified, it uses the parameter value explicitely
$ grep -nr functinality nledger
nledger/Source/NLedger.Tests/Utility/ServiceAPI/ServiceAPIIntegrationTests.cs:213:            // It means that queried objects (posts in this example) are accessible, but provide limited functinality.
nledger/Source/NLedger.Tests/Utility/ServiceAPI/ServiceAPIIntegrationTests.cs:220:            // (at least, for a limited scoope like "using" below). In bounds of this scope, all NLedger functinality is available.
$ grep -nr impicit nledger
nledger/Source/NLedger/Extensibility/Export/ListAdapter.cs:19:    /// that might identify the origin as a list. Replacing lists with the wrapper prevents impicit conversion by connectors that causes loosing
$ grep -nr indicater nledger
nledger/Contrib/NLManagement/NLCommon.psm1:156:# Helper method that print a structure indicater either success or failure
$ grep -nr lengthes nledger
nledger/Source/NLedger/Utility/AnsiTextWriter.cs:264:                        throw new InvalidOperationException("Unbalanced sequence lengthes");
$ grep -nr neccessary nledger
nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs:74:                        // If it is neccessary to troubleshoot this step, it is recommended to extract a test result object and go through detected errors
$ grep -nr occured nledger
nledger/Source/NLedger/Abstracts/Impl/PagerProvider.cs:84:                Logger.Current.Debug("pager", () => String.Format("Error occured during pager execution: {0}", ex.Message));
$ grep -nr partucular nledger
nledger/Source/NLedger/Commodities/Commodity.cs:36:         * In partucular, its member "amounts" (typedef std::map<commodity_t *, amount_t> amounts_map)
$ grep -nr priviledges nledger
nledger/Build/NLedgerBuild.ps1:156:     write-host -NoNewline "   all build steps with admin priviledges (testing is faster because of possibility to create native images);`r`n PS> "
nledger/Build/NLedgerBuild.ps1:164:     write-host -NoNewline "           compile, create a package and run Ledger tests. Does not require admin priviledges (longer testing time);`r`n PS> "
nledger/Build/NLedgerBuild.ps1:175:     write-host -NoNewline "`r`nNote: some build targets require administrative priviledges because they try to create NLedger native images by means of NGen."
nledger/Contrib/NLManagement/NLSetup.Console.ps1:40:  Write-Console "{c:gray}you must have administrative priviledges. Run this console as an administrator."
nledger/Contrib/NLManagement/NLSetup.Console.ps1:243:    Default: False. Indicates that the current scope is Application. You must have adminidstrative priviledges.
nledger/Contrib/NLManagement/NLSetup.Console.ps1:290:    Default: False. Indicates that the current scope is Application. You must have adminidstrative priviledges.
nledger/Contrib/Install/NLedger.Launcher.ps1:1:# This scripts runs the installer with elevated priviledges and keeps PS console open to let people read the status of installation activities.
nledger/Contrib/Install/NLedger.Install.ps1:10:         dramatically improves performance. Note: this action requires administrative priviledges.
$ grep -nr proviledges nledger
nledger/Build/NLedgerBuild.ps1:154:     write-host -NoNewline "            all build steps (compile, test, package). Does not require admin proviledges (longer build time);`r`n PS> "
nledger/Build/NLedgerBuild.ps1:166:     write-host -NoNewline "  compile, create a package and run Ledger tests. Requires admin proviledges (faster testing time);`r`n PS> "
$ grep -nr retrns nledger
nledger/Contrib/Install/SysPath.psm1:54:    Checks whether two paths are equal and retrns True or False. Properly manages relative and do-normalized paths.
$ grep -nr routinse nledger
nledger/Source/NLedger/Balance.cs:651:         * user.  Mostly used by `print' and other routinse where the sort
$ grep -nr scoope nledger
nledger/Source/NLedger.Tests/Utility/ServiceAPI/ServiceAPIIntegrationTests.cs:220:            // (at least, for a limited scoope like "using" below). In bounds of this scope, all NLedger functinality is available.
$ grep -n successfull nledger/Source/NLedger/Abstracts/IManPageProvider.cs
25:        /// <returns>Indicates whether the operation is successfull</returns>
$ grep -n successfull nledger/Source/NLedger.Tests/Scopus/OptionTests.cs
392:            Assert.Equal("true", option1.Value); // Indicates that the call was successfull
429:            Assert.True(option1.Handled); // Indicates that the call was successfull
$ grep -nr supresses nledger
nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs:68:                        // exit=False - supresses exiting the current process when all tests are done
nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs:69:                        // module=None - supresses module importing (basically, "main")
$ grep -nr valud nledger
nledger/Source/NLedger/Utility/BigValues/BigRational.cs:370:            // Round value and get integral and fractional parts of this valud
$ 
```

**Comment by spaette** [comment]
the following is authorised for use and/or modification by the nledger developers

```
$ cat nledger.sh
#!/bin/sh

sed -i "s/Assset/Asset/g" nledger/Contrib/test/regress/89233B6D-a.dat
sed -i "s/Assset/Asset/g" nledger/Contrib/test/regress/89233B6D.test
sed -i "s/Assset/Asset/g" nledger/Contrib/test/regress/89233B6D-b.dat
sed -i "s/Enuemrable/Enumerable/g" nledger/Source/NLedger/Extensibility/SessionExtensions.cs
sed -i "s/Optinal/Optimal/g" nledger/Source/NLedger/Utility/Settings/CascadeSettings/Sources/EnvironmentVariablesSettingsSource.cs
sed -i "s/Powershel /Powershell /g" nledger/Build/NLedgerWix/InstallerScripts.vbs
sed -i "s/Qunatity/Quantity/g" nledger/Source/NLedger.Tests/BigIntTests.cs
sed -i "s/Simluate/Simulate/g" nledger/Contrib/Extras/cat.cmd
sed -i "s/Uniinstalling/Uninstalling/g" nledger/Contrib/Install/NLInstall.psm1
sed -i "s/Wiindows/Windows/g" nledger/Contrib/Install/NLInstall.psm1
sed -i "s/Wiindows/Windows/g" nledger/Contrib/Install/SysPath.psm1
sed -i "s/Wiindows/Windows/g" nledger/Contrib/Install/SysHardLink.psm1
sed -i "s/accorrding/according/g" nledger/Source/NLedger.Tests/Amounts/AmountTests.cs
sed -i "s/addiing/adding/g" nledger/get-nledger-tools.ps1
sed -i "s/adminidstrative/administrative/g" nledger/Contrib/NLManagement/NLSetup.Console.ps1
sed -i "s/alligned/aligned/g" nledger/Contrib/test/regress/370-period.test
sed -i "s/amount'ss/amount's/g" nledger/Source/NLedger/Amounts/Amount.cs
sed -i "s/amount'ss/amount's/g" nledger/Contrib/test/regress/25A099C9.dat
sed -i "s/colorizatiion/colorization/g" nledger/Contrib/NLManagement/NLCommon.psm1
sed -i "s/coontains/contains/g" nledger/get-nledger-tools.ps1
sed -i "s/descructor/destructor/g" nledger/Source/NLedger/Scopus/Report.cs
sed -i "s/efficiemt/efficient/g" nledger/nledger.md
sed -i "s/excluse/exclusive/g" nledger/Source/NLedger/MainApplicationContext.cs
sed -i "s/expact/expect/g" nledger/Source/NLedger/Abstracts/IVirtualConsoleProvider.cs
sed -i "s/explicitely/explicitly/g" nledger/Source/NLedger.Extensibility.Python.Module/src/ledger/__init__.py
sed -i "s/explicitely/explicitly/g" nledger/Source/NLedger.Extensibility.Python.Module/build.ps1
sed -i "s/explicitely/explicitly/g" nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs
sed -i "s/explicitely/explicitly/g" nledger/Contrib/Python/GetPythonEnvironment.ps1
sed -i "s/functinality/functionality/g" nledger/Source/NLedger.Tests/Utility/ServiceAPI/ServiceAPIIntegrationTests.cs
sed -i "s/impicit/implicit/g" nledger/Source/NLedger/Extensibility/Export/ListAdapter.cs
sed -i "s/indicater/indicator/g" nledger/Contrib/NLManagement/NLCommon.psm1
sed -i "s/lengthes/lengths/g" nledger/Source/NLedger/Utility/AnsiTextWriter.cs
sed -i "s/neccessary/necessary/g" nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs
sed -i "s/occured/occurred/g" nledger/Source/NLedger/Abstracts/Impl/PagerProvider.cs
sed -i "s/partucular/particular/g" nledger/Source/NLedger/Commodities/Commodity.cs
sed -i "s/priviledges/privileges/g" nledger/Build/NLedgerBuild.ps1
sed -i "s/priviledges/privileges/g" nledger/Contrib/NLManagement/NLSetup.Console.ps1
sed -i "s/priviledges/privileges/g" nledger/Contrib/Install/NLedger.Launcher.ps1
sed -i "s/priviledges/privileges/g" nledger/Contrib/Install/NLedger.Install.ps1
sed -i "s/proviledges/privileges/g" nledger/Build/NLedgerBuild.ps1
sed -i "s/retrns/returns/g" nledger/Contrib/Install/SysPath.psm1
sed -i "s/routinse/routines/g" nledger/Source/NLedger/Balance.cs
sed -i "s/scoope/scope/g" nledger/Source/NLedger.Tests/Utility/ServiceAPI/ServiceAPIIntegrationTests.cs
sed -i "s/successfull/successful/g" nledger/Source/NLedger/Abstracts/IManPageProvider.cs
sed -i "s/successfull/successful/g" nledger/Source/NLedger.Tests/Scopus/OptionTests.cs
sed -i "s/supresses/suppresses/g" nledger/Source/NLedger.Extensibility.Python.Tests/PythonModuleTests.cs
sed -i "s/valud/value/g" nledger/Source/NLedger/Utility/BigValues/BigRational.cs
$ 
```

**Comment by dmitry-merzlyakov** [comment]
Hi @spaette , sorry for delated response.

Yes, please - current fixes are accumulated in [0.8.5](https://github.com/dmitry-merzlyakov/nledger/tree/next-dev-0.8.5) branch, so if you can create a PR for it. 

Also, let me know when you want to have them published in Master. I basically wanted to include .Net 6 support, but I am still busy with other activities, so I can target them to the next minor version and publish 0.8.5 with fixes only. Let me know your preferences.

Thank you!

**Comment by spaette** [comment]
PR submitted for next-dev-0.8.5.

I am not in a production environment.

Regarding eventual inclusion of .Net 6 support and when a release would include this Issue's typo fixes 

 _Es pressiert nicht. Wir haben Zeit_.

from Austrian translated as "It's not pressing. We have time."
